bigbone/build.gradle

```diff
@@ -1,6 +1,11 @@
 plugins {
     id 'bigbone.library-conventions'
     id 'bigbone.integrationtest-conventions'
+    id "org.jlleitschuh.gradle.ktlint" version "11.6.1"
+}
+
+allprojects {
```

Comment:
This is most probably no longer necessary, once above changes are implemented (kotlin conventions, version catalog).

Reply:
I had to add _alias(libs.plugins.ktlint)_ in the build.gradle both of the bigbone and bigbone-rx and sample-kotlin to make the project build rightly